rc/lsp.kak

```diff
@@ -2135,9 +2135,9 @@ define-command lsp-diagnostic-lines-disable -params 1 -docstring "lsp-diagnostic
 } -shell-script-candidates %{ printf '%s\n' buffer global window }
 
 define-command lsp-inlay-diagnostics-enable -params 1 -docstring "lsp-inlay-diagnostics-enable <scope>: Enable inlay diagnostics highlighting for <scope>" %{
-    add-highlighter "%arg{1}/lsp_inlay_diagnostics" replace-ranges lsp_inlay_diagnostics
+    add-highlighter "%arg{1}/lsp_inlay_diagnostics" flag-lines -after Default lsp_inlay_diagnostics
 	hook %arg{1} -group lsp-inlay-diagnostics ModeChange (push|pop):.*:insert "remove-highlighter %arg{1}/lsp_inlay_diagnostics"
-	hook %arg{1} -group lsp-inlay-diagnostics ModeChange (push|pop):insert:.* "add-highlighter %arg{1}/lsp_inlay_diagnostics replace-ranges lsp_inlay_diagnostics"
+	hook %arg{1} -group lsp-inlay-diagnostics ModeChange (push|pop):insert:.* "add-highlighter %arg{1}/lsp_inlay_diagnostics flag-lines -after Default lsp_inlay_diagnostics"
```

Comment:
we actually no longer need to hide it in insert mode, that was only motivated by the glitches which have been fixed. will do